jobs/index.js

```diff
@@ -153,7 +169,7 @@ function jobsClient(options) {
       var topicTokens = topic.split('/');
 
       // If not a job topic emit to application and return
-      if (!isJobTopic(topicTokens)) {
+      if (!isJobTopic(topicTokens) && !isThingShadowTopic(topicTokens) && !isThingShadowTopic(topicTokens)) {
```

Comment:
Is that a copy&paste error or am I missing a difference between the last two conditions?

Reply:
Would it make sense to first check whether the thing shadow module is really in use? Otherwise the original handler should apply, shouldn't it?